packages/poppler.rb

```diff
@@ -4,35 +4,41 @@ class Poppler < Package
   description 'Poppler is a PDF rendering library based on the xpdf-3.0 code base.'
   homepage 'https://poppler.freedesktop.org/'
   @_ver = '21.03.0'
-  version @_ver
+  version "#{@_ver}-1"
   license 'GPL-2'
   compatibility 'all'
   source_url "https://poppler.freedesktop.org/poppler-#{@_ver}.tar.xz"
   source_sha256 'fd51ead4aac1d2f4684fa6e7b0ec06f0233ed21667e720a4e817e4455dd63d27'
 
   binary_url({
-    aarch64: 'https://dl.bintray.com/chromebrew/chromebrew/poppler-21.03.0-chromeos-armv7l.tar.xz',
-     armv7l: 'https://dl.bintray.com/chromebrew/chromebrew/poppler-21.03.0-chromeos-armv7l.tar.xz',
-       i686: 'https://dl.bintray.com/chromebrew/chromebrew/poppler-21.03.0-chromeos-i686.tar.xz',
-     x86_64: 'https://dl.bintray.com/chromebrew/chromebrew/poppler-21.03.0-chromeos-x86_64.tar.xz'
+    aarch64: 'https://dl.bintray.com/chromebrew/chromebrew/poppler-21.03.0-1-chromeos-armv7l.tar.xz',
+     armv7l: 'https://dl.bintray.com/chromebrew/chromebrew/poppler-21.03.0-1-chromeos-armv7l.tar.xz',
+       i686: 'https://dl.bintray.com/chromebrew/chromebrew/poppler-21.03.0-1-chromeos-i686.tar.xz',
+     x86_64: 'https://dl.bintray.com/chromebrew/chromebrew/poppler-21.03.0-1-chromeos-x86_64.tar.xz'
   })
   binary_sha256({
-    aarch64: 'a4cb106683c7b8d289553fe9073201329619519f30ecc623dc6d08dbfd4eccb7',
-     armv7l: 'a4cb106683c7b8d289553fe9073201329619519f30ecc623dc6d08dbfd4eccb7',
-       i686: '54adf5a92963371ef36fbfa9078554b8539877bb036569d4f1b5d8627f7d2ddb',
-     x86_64: '4133f3f21f6ed0a9d6530dbe4d4d5467984958479e69f56061c25e2ff42fb4dc'
+    aarch64: '97b54286c9ff1d7c041ea465e586f797a3bf3e426ce16fc39b753feef2380161',
+     armv7l: '97b54286c9ff1d7c041ea465e586f797a3bf3e426ce16fc39b753feef2380161',
+       i686: 'b2bee26bfa243a2eb3290b9784b619215b0d46e0edf802a0088a96944ef6caa1',
+     x86_64: 'b1be0800a8af1881d28ad9c4f7f6ccabf351155d39787717da5a834f3aec59e8'
   })
 
   depends_on 'boost'
   depends_on 'cairo'
+  depends_on 'fontconfig'
   depends_on 'freeglut'
+  depends_on 'freetype'
+  depends_on 'glib'
   depends_on 'harfbuzz'
   depends_on 'lcms'
   depends_on 'libjpeg'
   depends_on 'libpng'
   depends_on 'libtiff'
+  depends_on 'nspr'
+  depends_on 'nss'
   depends_on 'openjpeg'
```

Comment:
Does openjpeg conflict with the libjpeg libraries?

Reply:
nope